easybuild/easyconfigs/f/FigureGen/FigureGen-51-foss-2019a.eb

```
@@ -0,0 +1,39 @@
+easyblock = 'CmdCp'
+
+name = 'FigureGen'
+version = '51'
+
+homepage = 'https://ccht.ccee.ncsu.edu/figuregen-v-49/'
+description = """  FigureGen is a Fortran program that creates images for ADCIRC files.
+It reads mesh files (fort.14, etc.), nodal attributes files (fort.13, etc.) and output 
+files (fort.63, fort.64, maxele.63, etc.). It plots contours, contour lines, and vectors. 
+Using FigureGen, you can go directly from the ADCIRC input and output files to a 
+presentation-quality figure, for one or multiple time snaps.  """
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [{'download_filename': 'master.zip', 'filename': SOURCE_ZIP}]
+checksums = ['4199816638cf95ccd661eb0509c46d80c9dae986b2053a2db23cce293dcfa328']
+
+github_account = 'ccht-ncsu'
```

One minor suggestion, please declare the `github_account` variable before its use in `GITHUB_LOWER_SOURCE` so that it is easier to read the easyconfig
```suggestion
github_account = 'ccht-ncsu'
source_urls = [GITHUB_LOWER_SOURCE]
sources = [{'download_filename': 'master.zip', 'filename': SOURCE_ZIP}]
checksums = ['4199816638cf95ccd661eb0509c46d80c9dae986b2053a2db23cce293dcfa328']

```